### ISSUE TITLE
TypeError: (0 , s.AI) is not a function

### ISSUE THREAD

**Issue by jahirfiquitiva** [aidenybai/million#963]
### What version of `million` are you using?

3.0.3

### Are you using an SSR adapter? If so, which one?

Vercel

### What package manager are you using?

yarn

### What operating system are you using?

Mac

### What browser are you using?

Arc

### Describe the Bug

During build, I get this error:

```gradle
 ⚠ Using edge runtime on a page currently disables static generation for that page
   Collecting page data  .TypeError: (0 , s.AI) is not a function
    at 90141 (/Users/jahir/dev/jahir/website/.next/server/chunks/144.js:492:873)
    at t (/Users/jahir/dev/jahir/website/.next/server/webpack-runtime.js:1:127)
    at 17222 (/Users/jahir/dev/jahir/website/.next/server/chunks/144.js:378:423)
    at Function.t (/Users/jahir/dev/jahir/website/.next/server/webpack-runtime.js:1:127)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async collectGenerateParams (/Users/jahir/dev/jahir/website/node_modules/next/dist/build/utils.js:919:21)
    at async /Users/jahir/dev/jahir/website/node_modules/next/dist/build/utils.js:1138:17
    at async Span.traceAsyncFn (/Users/jahir/dev/jahir/website/node_modules/next/dist/trace/trace.js:151:20)

> Build error occurred
Error: Failed to collect page data for /_not-found
    at /Users/jahir/dev/jahir/website/node_modules/next/dist/build/utils.js:1258:15
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  type: 'Error'
}
```

not sure what could be causing it

### What's the expected result?

Build completed successfully

### Link to Minimal Reproducible Example

https://github.com/jahirfiquitiva/jahir.dev/pull/73

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by Aslemammad** [comment]
Getting this error after cloning and pnpm dev: 

![image](https://github.com/aidenybai/million/assets/37929992/a1c1b9d8-79ce-4cb5-a26f-8f4145175203)

**Comment by Aslemammad** [comment]
I'm getting so many more errors like all the images in the /assets/images/ dir not being resolved, or during the build I get this 

![image](https://github.com/aidenybai/million/assets/37929992/266536b0-be0c-4bcb-a4d9-260a27d9b790)

Anyway, the issue you're facing would be fixed by #959

**Comment by jahirfiquitiva** [comment]
@Aslemammad it seems as if it wasn't taking any of the paths aliases set up in `tsconfig.json` ... so weird
